cli/src/main/java/org/jboss/pnc/bacon/cli/App.java

```diff
@@ -100,6 +100,17 @@ public static void setVerbosityIfPresent(boolean verbose) {
         }
     }
 
+    /**
+     * Make logback quiet if quiet flag is set
+     */
+    @Option(names = { "-q", "--quiet" }, description = "Silent output", scope = INHERIT)
+    public static void setQuietIfPresent(boolean quiet) {
+
+        if (quiet) {
+            ObjectHelper.setRootLoggingLevel(Level.OFF);
```

Comment:
What about setting it to Level.WARN ? Or even Level.ERROR ? (which matches what Maven does) i.e. disable INFO logging to be 'quiet? I think turning everything off is a bit too extreme?

Reply:
quiet is quiet :)

I guess Level.ERROR sounds about right.

Reply:
Thanks Nick for the input! I changed it to ERROR now. Good call! :)